src/views/BlogPage.vue

```diff
@@ -24,4 +24,9 @@ export default {
 
 <style scoped>
 
+.blog-wrapper {
+	padding-left: 150px;
```

Comment:
I would prefer a percentage here, otherwise it will look weird on mobile.
![image](https://user-images.githubusercontent.com/41529410/117393619-86007f00-aec2-11eb-9acb-8a03990f5803.png)
Also, make it padding for both left and right :)